### ISSUE TITLE
Gnome 40 compatibility

### ISSUE THREAD

**Issue by jdleslie** [do-sch/gnome-shell-touchpad-window-move#13]
`prefs.json` needs to be updated, but otherwise this extension works great with Gnome 40 with an update to `metadata.json`. Can you update for Gnome 40?

**Comment by do-sch** [comment]
That's pretty interesting. When I activate the extension, the standard GNOME gestures continue to be performed. Did you somehow disable the standard gestures?

Since GNOME has been using three-finger gestures since version 40 in a quite intuitive way, I actually had no plans to develop the extension any further. But if it is possible without a lot of effort, I would of course maintain it anyway.

**Comment by jdleslie** [comment]
Good point. I had the three finger gestures remapped to four fingers already via [icedman/gnome-shell-hammer](https://github.com/icedman/gnome-shell-hammer), which is how I found this extension.

**Comment by chumariesco** [comment]
Just updated to gnome 42.5. The extension doesn't work anymore.

Error: No property margin on GtkGrid
